package.json

```diff
@@ -100,7 +100,7 @@
     "mapbox-gl": "^1.13.0",
     "marked": "^2.0.1",
     "mini-css-extract-plugin": "^1.3.9",
-    "node-fetch": "^2.6.1",
```

Comment:
why did you remove this dependency?

Reply:
I was using it initially in the unit test to use the API, then reverted my changes when I extracted test data.  I didn't notice that it was used elsewhere.  Will revert.